filters/template/templatefilter.go

```diff
@@ -45,28 +45,23 @@ func (filter *TemplateFilter) Description() string {
 }
 
 // Passes returns true when the Filter matched the data.
-func (filter *TemplateFilter) Passes(data interface{}, value interface{}) bool {
-	switch v := value.(type) {
-	case string:
-		var res bytes.Buffer
+func (filter *TemplateFilter) Passes(data map[string]interface{}, v string) bool {
+	var res bytes.Buffer
 
-		if strings.Index(v, "{{test") >= 0 {
-			v = strings.Replace(v, "{{test", "{{if", -1)
-			v += "true{{end}}"
-		}
-
-		tmpl, err := template.New("_" + v).Funcs(templatehelper.FuncMap).Parse(v)
-		if err == nil {
-			err = tmpl.Execute(&res, data)
-		}
-		if err != nil {
-			panic(err)
-		}
+	if strings.Contains(v, "{{test") {
```

Comment:
Also, here I changed the `strings.Index` to `strings.Contains` because it is the same but is faster. Thanks to `go-simple`.

Reply:
(everything else here is just changed identation)